package.json

```diff
@@ -95,7 +97,7 @@
   },
   "dependencies": {
     "@lifeomic/chromicons-native": "^2.14.0",
-    "@react-native-community/picker": "^1.8.1",
+    "@react-native-picker/picker": "^2.4.9",
```

Comment:
Can you say more about this switch and why it is needed?  The typing additions make sense, but I'm not clear on this part.

Reply:
Yes, track-tile-ui used `Picker` from `react-native` for android but also used `PickerIOS` from `@react-native-community/picker`. 

The newer React Native used in this SDK no longer ships with `Picker` because it was moved to `@react-native-community/picker`.  Upgrading to that community version caused tests to break, leading me to discover that the library has been renamed to `@react-native-picker/picker`.

![Screenshot 2023-03-27 at 9 24 34 PM](https://user-images.githubusercontent.com/220893/228102598-22b50090-7f65-4576-9499-456b434309c3.png)

Now both `Picker` and `PickerIOS` are imported from this same lib.